.github/workflows/docker.yml

```diff
@@ -22,15 +22,15 @@ jobs:
     # This is done to limit the runner cost.
     if: github.event.pull_request.draft == false
     name: Build images and push to GHCR
-    runs-on: buildjet-8vcpu-ubuntu-2204
-    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
```

Comment:
Are you positive that a clean build on `ubuntu-latest` will not timeout on this? E.g., when the toolchain is updated?

Reply:
Note that cross-compilation for ARM takes _a lot_ of time, did you verify the build with it?

Reply:
Hmm, I'm not sure actually. But a slow clean build seems to be a deal breaker, should we stick on buildjet?

Reply:
> Note that cross-compilation for ARM takes a lot of time, did you verify the build with it?

@LesnyRumcajs Yes the CI job for PR is targeting arm64, which takes ~18min on GH-hosted runner with cache.

```yaml
# Compile Docker image only for ARM64 for a regular PR to save some CI time.
platforms: ${{ (github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/v')) && 'linux/amd64,linux/arm64' || 'linux/arm64' }}
```

Reply:
Could you please show me a build where an `arm64` image was built from scratch?

Reply:
If we are not using buildjet then it's fine to have a generous timeout - we don't pay for it (though we occupy the workers which are limited)

Reply:
> Could you please show me a build where an arm64 image was built from scratch?

@LesnyRumcajs I don't have one yet, and I don't think it will be fast enough for a job that is enabled for every PR, otherwise we'd already use GH-hosted runner : )  And since this is a valid case, should we switch back to buildjet as sccache could still save some less $ for buildjet . Does that make sense?

Reply:
Let's first get some numbers:
1. Clean build for `amd64` on `ubuntu-latest`. Reasoning: we want to know the worst-case scenario for a regular build on a branch.
2. Consecutive build for `amd64`.
3. Clean build for `arm64` on `ubuntu-latest`. 
4. Consecutive builds for `arm64`.

Then let's compare it with the current state-of-the-art.

A possible solution, to use something from both worlds, is to use the GH worker for PRs and buildjet for builds involving cross-compilation.

Reply:
Does it make sense?